dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java

```diff
@@ -1351,7 +1483,11 @@ public int compare(MetadataValue mdv1,MetadataValue mdv2) {
 
                 if (hitHighlightingFields.contains(field) || hitHighlightingFields
                     .contains("*") || hitHighlightingFields.contains(unqualifiedField + "." + Item.ANY)) {
-                    doc.addField(field + "_hl", value);
+                    if (authority != null) {
+                        doc.addField(field + "_hl", value + "###" + authority);
```

Comment:
Should the `"###"` be turned into a constant?  It looks like it must be a field separator, which implies it likely should be a constant.

Reply:
yes, we already have such constant. Reused